src/main/java/frc/robot/commands/autos/AutoScoreTwo.java

```diff
@@ -0,0 +1,71 @@
+package frc.robot.commands.autos;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.commands.FollowPathWithEvents;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.commands.IntakeSequence;
+import frc.robot.commands.autos.components.AutoScoreOne;
+import frc.robot.subsystems.Lights;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.drive.DriveCal;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.grabber.Grabber;
+import frc.robot.utils.ScoringLocationUtil.ScoreHeight;
+
+import java.util.HashMap;
+
+public class AutoScoreTwo extends SequentialCommandGroup {
```

Comment:
Maybe clarify that this is for the open side, either in the name or as a /* */ comment?